src/__tests__/tigris.rpc.spec.ts

```diff
@@ -348,6 +349,34 @@ describe('rpc tests', () => {
 		});
 	});
 
+	it('search', (done) => {
+		const tigris = new Tigris({serverUrl: '0.0.0.0:' + SERVER_PORT});
+		const db3 = tigris.getDatabase('db3');
+		let bookCounter = 0;
+		let success = true;
+		db3.getCollection('books')
+			.search({
+				q: 'philosophy',
+				facetQuery: {
+					tags: createFacetQueryOptions()
+				}
```

Comment:
add more test that exercises all types of supported search queries.

Reply:
Create a new test suite in `tests/search`